src/modules/NtpClient.rb

```diff
@@ -1045,42 +1045,58 @@ def Summary
       summary
     end
 
+
     # Test if specified NTP server answers
     # @param [String] server string host name or IP address of the NTP server
+    # @return [Boolean] true if NTP server answers properly
+    def reachable_ntp_server?(server)
+      # testing the server using IPv4 and then using IPv6 protocol
+      # bug #74076, Firewall could have been blocked IPv6
+      return true if sntp_test(server) == 0
+
+      sntp_test(server, 6) == 0
+    end
+
+    # -K /dev/null: use /dev/null as KoD history file (if not specified,
+    #               /var/db/ntp-kod will be used and it doesn't exist)
+    # -c: concurrently query all IPs; -t 5: five seconds of timeout
+    # @param [String] server string host name or IP address of the NTP server
+    # @param [Fixnum] integer ip version to use (4 or 6)
+    # @return [Fixnum] sntp exit code or 1 in case of lookup error
+    def sntp_test(server, ip_version = 4)
+      output = SCR.Execute(
+        path(".target.bash_output"),
+        "/usr/sbin/sntp -#{ip_version} -K /dev/null -t 5 -c #{server}"
+      )
+
+      Builtins.y2milestone("sntp test response: #{output}")
+      # sntp returns always 0 if not called with option -S or -s (set system time)
+      # so this is a workaround at least to return 1 in case server is not
+      # reachable
+      return 1 if output["stderr"].include?("lookup error")
```

Comment:
To check the content of the output you should first ensure it will be in English appending `LANG=C` to the command line call.